src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs

```diff
@@ -136,20 +136,5 @@ public void Should_TryParse_Return_True(string cultureName)
             Assert.IsTrue(parsed);
             Assert.AreEqual(tickMessage, _expectedMessage);
         }
-
-        [Test, TestCaseSource(typeof(CultureNameTestCase), nameof(CultureNameTestCase.CultureNames))]
-        public void Should_TryParse_Protocol60_Return_True(string cultureName)
```

Comment:
I don't understand why you're getting rid of this test. It tests something different to the others.

Reply:
well, this one should technically only be used when requesting new data. Older data should necessary come from disk.

Reply:
So we should move the test to a different set? HistoricalFileFacadeTests? It's still worth testing somewhere though, isn't it?

Reply:
well, in fact, ill revert back most my changes for TickMessages, yours seems totally okay.